app/models/slack_user.rb

```diff
@@ -1,3 +1,3 @@
 class SlackUser < ApplicationRecord
-  validates :slack_id, :slack_name, :slack_real_name, :slack_email, :user_id, presence: true
+  validates :slack_id, :slack_name, :slack_real_name, :slack_email, presence: true
```

Comment:
Removed the user_id from the validation since a slack_user has the possibility of being set without a user being defined in the main system